CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [#4568](https://github.com/bbatsov/rubocop/pull/4568): Fix autocorrection for `Style/TrailingUnderscoreVariable`. ([@smakagon][])
 * [#4586](https://github.com/bbatsov/rubocop/pull/4586): Add new `Performance/UnfreezeString` cop. ([@pocke][])
 * [#2976](https://github.com/bbatsov/rubocop/issues/2976): Add `Whitelist` configuration option to `Style/NestedParenthesizedCalls` cop. ([@drenmi][])
+* [#4368](https://github.com/bbatsov/rubocop/issues/4368): Extend flexibility of `Performance/HashEachMethod` cop. ([@gohdaniel15][])
```

Comment:
Try to outline what was changed in the description. E.g.: "Make `Performance/HashEachMethod` inspect all send nodes."